Hypnotic-Herons/Sprint-2
# Employee template

## Description of Proposed Changes
- Corrected some indentation issues and references in the existing model and view files
- Made some adjustments for the addition of training program files (some merged stuff got messed up)


## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout employeeTemplate
code .

```


1. python manage.py makemigrations HRIT
1. python manage.py migrate
1. python manage.py createsuperuser (this is just for your local use)
1. login on the admin view http://127.0.0.1:8000/admin
1. add a department
1. add a few employees
1. look at the browser view for employees http://127.0.0.1:8000/HRIT/employees/
1. you should see the employees you just entered

  

## Impacted Areas in Application

Database 


## Mentions @username

@Megraohoh @jessnswift 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
